File: app/src/main/java/com/babestudios/companieshouse/data/model/officers/appointments/Appointments.java
package com.babestudios.companieshouse.data.model.officers.appointments;


import com.babestudios.companieshouse.data.model.officers.DateOfBirth;
import com.babestudios.companieshouse.data.model.officers.OfficersLinks;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Appointments {
	@SerializedName("date_of_birth")
	public DateOfBirth dateOfBirth;
	@SerializedName("etag")
	public String etag;
	@SerializedName("is_corporate_officer")
	public String isCorporateOfficer;
	@SerializedName("items")
	public List<Appointment> items = null;
	@SerializedName("items_per_page")
	public String itemsPerPage;
	@SerializedName("kind")
	public String kind;
	@SerializedName("links")
	public OfficersLinks links;
	@SerializedName("name")
	public String name;
	@SerializedName("start_index")
	public String startIndex;
	@SerializedName("total_results")
	public String totalResults;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/officers/appointments/Appointments.kt
package com.babestudios.companyinfouk.data.model.officers.appointments


import com.babestudios.companyinfouk.data.model.common.MonthYear
import com.babestudios.companyinfouk.data.model.common.SelfLinkData
import com.google.gson.annotations.SerializedName

class Appointments {
	@SerializedName("date_of_birth")
	var dateOfBirth: MonthYear? = null
	@SerializedName("etag")
	var etag: String? = null
	@SerializedName("is_corporate_officer")
	var isCorporateOfficer: String? = null
	@SerializedName("items")
	var items: List<Appointment>? = null
	@SerializedName("items_per_page")
	var itemsPerPage: String? = null
	@SerializedName("kind")
	var kind: String? = null
	@SerializedName("links")
	var links: SelfLinkData? = null
	@SerializedName("name")
	var name: String? = null
	@SerializedName("start_index")
	var startIndex: String? = null
	@SerializedName("total_results")
	var totalResults: String? = null
}
